Gemfile

```diff
@@ -29,6 +29,7 @@ group :development do
   gem "binding_of_caller"
   gem "listen"
   gem "nokogiri"
+  gem "web-console"
```

Comment:
I thought it might be useful in the same way that byebug and pry are, and we don't expect people to install those themselves.

Reply:
I wouldn't expect anyone to know it's there to even find it to be useful. There's at least a common expectation that pry or byebug might be there (and even those are annoying jumping between projects as you try one and then the other to find which is installed).

Our gemfiles often seem to end up with lots of unused stuff and I'm of the view that unless you have strong evidence that it'll be used long term just don't add it permanently. If we take this repo we've got: better_error, binding_of_caller, minitest-focus that likely have no actual use, then pry **and** byebug. So slowly the junkyard builds up.

Reply:
`web-console` is used in multiple places in [alphagov](https://github.com/search?l=Ruby&p=1&q=org%3Aalphagov+web-console+language%3ARuby&ref=advsearch&type=Code) including [publishing-api](https://github.com/alphagov/publishing-api/blob/master/Gemfile#L32) so I don't think it's unreasonable to expect people to expect it's presence.

Reply:
I've never seen anyone use it in Publishing API over the years I was working on it - doesn't even seem to make any sense there since it's an API. So that example just looks like more Gemfile junk to me...

Reply:
Is that you volunteering to do an audit of the gem files? 😎

Reply:
ooh I loved the web-console! I removed the commit. Maybe we can add it as a separate PR at some point <3